includes/class-wc-order.php

```diff
@@ -885,10 +885,11 @@ public function get_formatted_shipping_address( $empty_content = '' ) {
 		$address = '';
 
 		if ( $this->has_shipping_address() ) {
-			$address = apply_filters( 'woocommerce_order_formatted_shipping_address', $this->get_address( 'shipping' ), $this );
-			$address = WC()->countries->get_formatted_address( $address );
+			$address = WC()->countries->get_formatted_address( $this->get_address( 'shipping' ) );
 		}
 
+		$address = apply_filters( 'woocommerce_order_formatted_shipping_address', $address, $this );
```

Comment:
This changes the filter data, since now will filter the formatted address instead of filtering before.

Reply:
I see. Made the changes.

However, with that change, even though I might change the address (via filter) to a completely different value and order has a shipping address, my value would be overridden. Unless shipping address 1 and 2 are empty.
Same goes for the billing address.

Reply:
Actually, this change is not OK. The `$address` value is an array from `WC_Order::get_address( 'shipping' )`, but if the order hasn't got shipping address, the function will return the same array. @claudiosanches maybe we should add another filter, or...?

Reply:
Yes, the current change still is bad, since will change the value if expect to be an empty string.
A new filter should be probably better.